### DISCUSSION TITLE
Enable TSC diagnostics for whole project in takeover mode

### DISCUSSION THREAD

**Discussion by h-sigma** [vuejs/language-tools#2436] [Q&A]
Hello,

Currently, volar only displays problems for files that are opened in vscode. I've tried looking it up, but I can't find how to enable diagnostics for the complete project regardless of what files are opened. I have plenty of ram to spare.

**Comment by johnsoncodehk** [comment]
This is an intentionally removed feature, because this feature is not specific to the vue project, you can consider implementing a VSCode extension that can support all languages ​​for this purpose.
